services/jitpack/jitpack-downloads.tester.js

```diff
@@ -1,23 +1,22 @@
 'use strict'
 
-const { ServiceTester } = require('../tester')
+const t = (module.exports = require('../tester').createServiceTester())
+const { isMetricOverTimePeriod } = require('../test-validators')
 
-const t = (module.exports = new ServiceTester({
-  id: 'JitpackDownloads',
-  title: 'JitpackDownloads',
-  pathPrefix: '/jitpack',
-}))
+t.create('weekly (github)')
+  .get('/dw/github/jitpack/maven-simple.json')
+  .timeout(10000)
```

Comment:
In my local testing, this API and these tests are all returning well within 500ms. However, I thought it best to include an increased timeout given the historical timeout issues we've had with the Jitpack service. Happy to remove them if anyone is opposed though